scenarios/hci/network_data.yaml.j2

```diff
@@ -0,0 +1,61 @@
+---
+- name: Storage
+  mtu: 1500
+  vip: true
+  name_lower: storage
+  dns_domain: storage.{{ cloud_domain }}.tld.
```

Comment:
remove the .tld bit too but leave the trailing '.' 

storage.{{ cloud_domain }}.

Reply:
good catch, fixed, thanks!